Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -194,15 +194,74 @@ class ConfigurationTests: XCTestCase {
     func testConfiguresCorrectlyFromDict() {
         let ruleConfiguration = [1, 2]
         let config = [RuleWithLevelsMock.description.identifier: ruleConfiguration]
-        let rules = testRuleList.configuredRules(with: config)
+        // swiftlint:disable:next force_try
+        let rules = try! testRuleList.configuredRules(with: config)
```

Comment:
I get an error: `Errors thrown from here are not handled` if I just use `try`. Should the `func` be marked as `throws`? I haven't found any test that is marked as `throws` 🤔